site2/docs/sql-getting-started.md

```diff
@@ -117,61 +116,3 @@ presto> select * from pulsar."public/default".generator_test;
 ```
 
 You can query the mock data.
-
-## Query your own data
```

Comment:
Confusing content. Removed.

It's possible to write a new example but we should remove confusing content at least.